code/style.css

```diff
@@ -1,3 +1,84 @@
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: #F7D4BC;
+  font-family: 'Gemunu Libre', sans-serif;
+}
+
+header {
+  text-align: center;
+  font-size: 30px;
+}
+
+.profile {
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  text-align:center;
+  color: #218380;
+  margin:auto;
+  width: 100%;
+  flex-direction: column;  
+}
+
+img {
+  border-radius: 50%;
+  width: 200px;
+  margin: 15px;
+  box-shadow: -30px 30px 30px rgba(0, 0, 0, 0.3);
```

Comment:
It's nice that you added a shadow to your profile image! the shadow looks a little bit "off" from the shape of the image, a suggestion is to play around a little with the value-numbers of the shadows. I also worked a while to figure out the numbers to make the shadow, its tricky! 😊